juju/controller.py

```diff
@@ -5,66 +5,61 @@
 from . import tag
 from . import utils
 from .client import client
```

Comment:
`from .client import client, connector` and remove the line below.

Reply:
done